src/ACF/BaseFields.php

```diff
@@ -769,5 +674,4 @@ public function generateUniquePrefix($prefix, $label)
     {
         return md5($prefix . strtolower(str_replace(' ', '_', $label)));
     }
```

Comment:
Have you just replaced this file with the one you had on your site with fields that were missing rather than adding in the new ones that were missing?

Not ideal as there are too many changes to this file to review what was originally there and what is new to see what is useful

Reply:
I added in the missing fields but also reordered them to they are grouped in the same way as the ACF docs. Could have left it as is, added the fields at the bottom if that is easier?